features/step_definitions/cluster_configuration/csi.rb

```diff
@@ -143,3 +143,16 @@
   step %Q/the step should succeed/
   step %Q/the "#{dynamic_pvc_name}" PVC becomes :bound within 120 seconds/
 end
+
+Given /^I check the env has clustercsidrivers$/ do
```

Comment:
Do you mean in each case we need to add this extra step? Generally for these clustercsidriversn not installed CI jobs we could filter out our cases. WDYT?

Reply:
Agree, I raised https://github.com/openshift/release/pull/44064.